django/utils/translation/trans_real.py

```diff
@@ -566,6 +566,8 @@ def templatize(src, origin=None):
     comment = []
     lineno_comment_map = {}
     comment_lineno_cache = None
+    # Adding the u prefix on Python 3 allows gettext to recognize the Unicode string (#26093).
```

Comment:
Maybe we could omit "on Python 3" so we don't have to edit the comment when dropping Python 2 support. Also wrap comment at 79 characters.